local-cli/server/server.js

```diff
@@ -12,6 +12,7 @@ const chalk = require('chalk');
 const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
+const findSymlinksPaths = require('./findSymlinksPaths');
```

Comment:
Also, it appears you're requiring the symlink resolver but you're not actually using it.